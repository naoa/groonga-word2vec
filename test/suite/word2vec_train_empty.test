register word2vec/word2vec

word2vec_train
