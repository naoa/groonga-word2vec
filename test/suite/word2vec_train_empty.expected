register word2vec/word2vec
[[0,0.0,0.0],true]
word2vec_train
